gnmvidispine/vs_search.py

```diff
@@ -60,28 +60,28 @@ def __init__(self,operation):
     def addCriterion(self,crit):
         if not isinstance(crit, dict): raise ValueError
 
-        for k,v in crit.items():
+        for k,v in list(crit.items()):
             self._criteria[k] = v
 
     def to_xml(self,parentNode):
         opNode = ET.SubElement(parentNode, 'operator', attrib={'operation': self.operation})
-        for k,v in self._criteria.items():
+        for k,v in list(self._criteria.items()):
             if isinstance(v,VSSearchOperator):
                 v.to_xml(opNode)
             else:
                 fieldEl = ET.SubElement(opNode,'field')
                 nameEl = ET.SubElement(fieldEl,'name')
-                nameEl.text = unicode(k)
+                nameEl.text = str(k)
```

Comment:
see earlier point about a safe_string function. I guess it is safe here as long as `k` is a non-byte string, I can't think of why it wouldn't be.

Reply:
Done.